langmeta-sqlite/src/main/scala/org/langmeta/sqlite/Main.scala

```diff
@@ -67,113 +68,119 @@ object Main {
 
           semanticdbFilenames.foreach { semanticdbFilename =>
             val path = Paths.get(semanticdbFilename)
-            val bytes = Files.readAllBytes(path)
-            val db = Database.parseFrom(bytes)
-            db.documents.foreach { document =>
-              documentsToPaths.get(document.filename) match {
-                case Some(existingPath) =>
-                  val what = document.filename
-                  val details = "both $existingPath and $path"
-                  println(s"Duplicate document filename $what in $details")
-                case None =>
-                  genuineDocuments += 1
-                  documentsToPaths(document.filename) = path
+            try {
```

Comment:
What sort of errors are you seeing?  I have no errors parsing semanticdbs, but I did find when I ran your dedup script on my laptop it did not filter down to semanticdb files specifically and threw errors on dsstore files.

Reply:
If you accidentally pass a non-semanticdb file to the app, there's no need to crash it completely. That file should be skipped instead. That's the error that I saw - nothing too fancy :)